ramls/inventory-view-instance.json

```diff
@@ -26,7 +22,6 @@
       }
     }
   },
-  "additionalProperties": false,
-  "required": ["instanceId", "instance"]
+  "additionalProperties": true
```

Comment:
Just a FYI that `instance.json` has some required fields.

Reply:
Thanks, I've added them to the schema and to the `required` section.